planning/behavior_path_planner_common/include/behavior_path_planner_common/utils/path_safety_checker/path_safety_checker_parameters.hpp

```diff
@@ -79,6 +80,16 @@ struct ExtendedPredictedObject
   autoware_auto_perception_msgs::msg::Shape shape;
   std::vector<autoware_auto_perception_msgs::msg::ObjectClassification> classification;
   std::vector<PredictedPathWithPolygon> predicted_paths;
+
+  ExtendedPredictedObject() = default;
+  explicit ExtendedPredictedObject(const PredictedObject & object)
+  : uuid(object.object_id),
+    initial_pose(object.kinematics.initial_pose_with_covariance),
+    initial_twist(object.kinematics.initial_twist_with_covariance),
+    initial_acceleration(object.kinematics.initial_acceleration_with_covariance),
```

Comment:
I think the `shape` is missing from the new constructor.
```suggestion
    shape(object.shape),
```